src/lang/en.json

```diff
@@ -320,6 +320,7 @@
   "tooltip_biggest_hit": "Largest instance of damage on a hero",
   "tooltip_damage_dealt": "Damage dealt to heroes by items/abilities",
   "tooltip_damage_received": "Damage received from heroes by items/abilities",
+  "tooltip_registered_user": "Registered user",
```

Comment:
Ye, but not now, 🍀  I'll plan to make some changes to player header that fixes this https://github.com/odota/ui/issues/81#issuecomment-251370349. Then I will add strings..